public/app/features/provisioning/RepositoryListPage.tsx

```diff
@@ -150,8 +150,8 @@ function getListURL(repo: Repository, stats: ResourceCount): string {
   if (stats.resource === 'playlists') {
```

Comment:
This page is loading forever if you create a repository with the sync disabled. We should probably hide the sync button. 

![image](https://github.com/user-attachments/assets/4bd1e06a-4e0e-4f2b-965f-b530e8b19b90)


![image](https://github.com/user-attachments/assets/327c5f05-7bfe-40ce-a6ba-9f2fa19b494b)

Reply:
it now sends you to the edit page...

Reply:
ok, spinner issue still remaining :)